### ISSUE TITLE
lost Expedition tab after editing expedition save :(

### ISSUE THREAD

**Issue by wkitty42** [goatfungus/NMSSaveEditor#1032]
[EDIT] this is a new save for the expedition, not an existing one that used the expedition terminal on the Anomaly[/EDIT]

i got tired of cheesing in Wiring Looms by reloading the save so i used 1.17.1 to give myself some... got back into the game and did a few more thing harvesting stuff before deciding to do the next milestone... was quite surprised to see the Expedition tab completely missing even though there are expedition tasks in my Log tab...

[time passes]

ok, i went back in to take a screenshot to add to this issue and the Expedition tab was back! WHEW! now i can claim the rewards for those parts i've done... talk about being scared the save was ruined... damn!

anyway, i'm leaving this issue in place so maybe GF can figure out the problem and fix it so others don't run into it and get freaked out like i did...

**Comment by perpetualyawn** [comment]
I think it's a game bug rather than editor. May have just been coincidence. My expedition tab disappeared while in game after heading into space from the starting planet. On the surface it was there, flew into space and it was gone.

**Comment by OGTeknight** [comment]
The editor converts the saved "Expedition" file into a "Normal" save file and removes all access to the expedition tab. I went through a series of steps to confirm it. You can tell just but how the symbol next to your save in the game changes from the expedition symbol, multi-ringed circular logo to the normal game symbol, Atlas logo.

**Comment by wkitty42** [comment]
to be clear, my save is a new dedicated save for the expedition... it is not an existing one that used the expedition terminal in the Anomaly to join the fun... i have edit my original above to clarify this...

**Comment by wkitty42** [comment]
> I think it's a game bug rather than editor. May have just been coincidence. My expedition tab disappeared while in game after heading into space from the starting planet. On the surface it was there, flew into space and it was gone.

yes, it may very well be a game bug... i didn't notice it immediately... i had been back in-game for 15-20 minutes, at least, before i noticed the Expedition tab was missing...

**Comment by OGTeknight** [comment]
I tested both brand new expedition and expedition through the Anomaly and I'm getting the same results, after saving through the editor, it turns my saves into normal games and the tab is gone, every time.  ¯\_(ツ)_/¯

**Comment by wkitty42** [comment]
ahhhh! i see what you are saying! on the save select screen, it is a "Community Expedition" but after the editor is used on it, it is "Normal"... i got it... interesting...

**Comment by OGTeknight** [comment]
> I experienced this issue also, but it is solved by saving in the game, exiting the game and reentering it. The 'Expedition' page becomes available again.

Thanks Shock, that worked! I repeated it several times, its solid. :D
